crates/client-api-messages/Cargo.toml

```diff
@@ -15,6 +15,7 @@ bytestring.workspace = true
 brotli.workspace = true
 chrono = { workspace = true, features = ["serde"] }
 enum-as-inner.workspace = true
+flate2.workspace = true
```

Comment:
Could be a follow-up, but I think we should consider feature-gating this (and brotli, I suppose). There's at least some value to reducing the Rust client SDK's dependency tree, especially since (as discussed in another comment) I see very little value for an external user of the SDK in switching away from the default compression format.